### PR TITLE
simplification of Envs for docker setup. Closes #199

### LINKED ISSUE #199: using env.docker does not read the right values for env variables
many tests fail with `Get error from return Badges NotAuthorizedError: Invalid token signature` when the environment file is named `env.docker` rather than `.env`.  `cp env.docker .env` is a temporary workaround.  @josmas thinks this might be related to how habitat is being used.

### PR DESCRIPTION
**Pull request by josmas**
As per title!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,7 +57,25 @@ If you would like to develop against the hosted custom badgekit-api we have runn
...
+
+
+#### Run locally
+
+* Install PaperBadger's Node dependencies: `npm install`
+* Make sure MongoDB and redis are running and locally accessible.
+* Setup your environment variables as explained in a previous section
 * Run `npm start`, and open up `http://localhost:5000/` in your favourite web browser!
 
 To run the application successfully you need to have [mongodb](https://www.mongodb.org/) server and [redis-server](http://redis.io/download) running locally. You can install these from their official website or use your favorite package manager.
```
Comment (drammock) [inline README.md]: suggest merging this text into line 78.

Comment (drammock) [inline README.md]: (err... 77.  The line that says `* Make sure MongoDB and redis are running and locally accessible.`)

**Comment by drammock** [comment]
other than the one minor README suggestion LGTM.

File: README.md (hunk 1 above)
Comment (josmas) [inline README.md]: Agree. It is redundant now that has been moved around. Will merge into a one liner.

**Comment by josmas** [comment]
@acabunoc I am going to merge this one because a couple of people in the Europe call mentioned they will give the project a go. It doesn't touch any of the real project codebase, so it can always be reverted easily.

**Comment by josmas** [comment]
rebased and merged.

**Comment by acabunoc** [comment]
good plan, I have no idea why my docker isn't working. I think I'll give up on that.

**Pull request closed without merging**